### PR TITLE
Router

### PR DESCRIPTION
**Pull request by ameentalahmeh**
add router function that calls handlers depends on `URL endpoint`

### CODE REVIEW

**Comment by mattlub** [comment]
the server is returning 404 for the css and js and favicon

screen shot 2018-12-11 at 14 41 38

**Comment by ameentalahmeh** [comment]
> the server is returning 404 for the css and js and favicon
> 
> screen shot 2018-12-11 at 14 41 38

the routes is `/css/main.css` and so on ...

**Comment by mattlub** [comment]
@ameentalahmeh you should fix it 😄

File: src/server/router.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//-- Import Modules
+const fs = require('fs')
+const path = require('path');
```
Comment (mattlub) [inline src/server/router.js]: fs and path are unused in this file. I am happy to merge, but I think you should set up eslint to catch cases like this.

Comment (mattlub) [reply src/server/router.js]: it would also fix the indentation in this file!

Comment (mattlub) [reply src/server/router.js]: let me know when you've seen this comment or when you've fixed it, and I'll merge.

Comment (ameentalahmeh) [reply src/server/router.js]: we fixed it, and we will create a new branch to define eslint.

**Comment by mattlub** [comment]
nice :)

**Pull request merged**